contrib/spec/podman.spec.in

```diff
@@ -203,6 +203,7 @@ BuildRequires: python3-varlink
 Requires: python3-setuptools
 Requires: python3-varlink
 Requires: python3-dateutil
+Requires: python3-psutil
```

Comment:
sgtm, just curious, is this something needed because of the MacOS porting?

Reply:
@lsm5 Yes, I needed to get more aggressive with killing off child processes.  Process groups have different defaults. I will also need this package for the Windows port.